### DISCUSSION TITLE
Transfer github Education to a second account

### DISCUSSION THREAD

**Discussion by YaroslavVorobyov** [community/community#51705] [New to GitHub]
Hello. I learned that a github account can have many emails. How can I change my account?

**Comment by MissieKahsay** [comment]
Hello @YaroslavVorobyov,
You can change the email address associated with your GitHub account by following these steps:

- Log in to your GitHub account.

- Click on your profile icon in the top right corner of the screen and select "Settings" from the dropdown menu.

- In the left sidebar, click on "Emails" to view the email addresses associated with your account.

- To add a new email address, click on the "Add email address" button and enter the new email address you want to use. You will need to verify the new email address by clicking on the verification link that will be sent to it.

- To make the new email address your primary email address, click on the "Make primary" button next to the email address in the list.

- To remove an email address from your account, click on the "Delete" button next to the email address in the list.

- If you're changing the email address associated with your GitHub account to one that you've already added to your account, make sure to verify the new email address by clicking on the verification link that will be sent to it.

Once you've changed your email address, GitHub will send a confirmation email to the new email address you provided. Be sure to check your email and click on the verification link to confirm the change.

Best,
